### PR TITLE
feat(nx): re-introduce @nrwl/bazel

### PR DESCRIPTION
**Pull request by FrozenPandaz**
## Current Behavior (This is the behavior we have today, before the PR is merged)

`@nrwl/bazel` is deprecated and does not work.

## Expected Behavior (This is the new behavior we can expect after the PR is merged)

A new version of `@nrwl/bazel` is added to the workspace which will utilize the Nx CLI through bazel.

The core functionality is to generate build files with knowledge from the dep graph and sync them for the user.

This feature is experimental.

## Issue
Related #1511

### CODE REVIEW

File: packages/bazel/src/schematics/sync/files/build-file/BUILD.bazel__tmpl__ [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    data = [
+        ":<%= project.name %>",
+        # Dependencies
+        <% for (dep of depGraph.dependencies[project.name]) { %>"//<%= depGraph.projects[dep.projectName].root %>:<%= dep.projectName %>",
+        <% } %>
+        # Root Files
+        "//:angular.json",
+        "//:package.json",
+        "//:tsconfig.json",
+        # NOTE: Use @npm//node_modules causes bundling to fail due to Babel errors.
```
Comment (vsavkin) [inline packages/bazel/src/schematics/sync/files/build-file/BUILD.bazel__tmpl__]: I thought that has been fixed. I'd prefer not to list individual packages.

File: packages/bazel/collection.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Nx Angular",
```
Comment (vsavkin) [inline packages/bazel/collection.json]: nx bazel

**Review by vsavkin — CHANGES_REQUESTED** [review]
Waiting to merge @jaysoo PR.

File: packages/bazel/src/schematics/sync/files/build-file/BUILD.bazel__tmpl__ (hunk 1 above)
Comment (FrozenPandaz) [reply packages/bazel/src/schematics/sync/files/build-file/BUILD.bazel__tmpl__]: @mrmeku has a fix for this.

**Comment by FrozenPandaz** [comment]
Closed in favor of https://github.com/nrwl/nx/pull/2127

If you would like to contribute to this feature, please feel free to make PRs into https://github.com/nrwl/nx/tree/bazel

**Pull request closed without merging**